hmri_create_b1map.m

```diff
@@ -220,6 +220,28 @@
 P    = b1map_params.b1input; % B1 data - 11 pairs
 Q    = b1map_params.b0input; % B0 data - 3 volumes
 
+% splitting images in echo1 and echo2 volumes
+% wold be nice to check that echo times are as expected
+P_e1 = P(1:2:end, :);
+P_e2 = P(2:2:end, :);
+
+% calc_SESTE_b1map expects fa in decreasing order
+[b1map_params.b1acq.beta, fa_order] = sort(b1map_params.b1acq.beta, ...

```

Comment:
Throw error if beta not the same length as the number of images.

Reply:
Done, made before splitting P into pairs on line 223.

Reply:
This should make the need to use "P(1:2:size(P_SE, 1)*2)" rather than "(1:2:end) redundant, too, right? If someone has accidentally added too many files (e.g. previously computed B1 maps), then it should crash on this check.